manifests/revoke.pp

```diff
@@ -58,7 +58,7 @@
   Openvpn::Revoke[$name]
 
   exec { "revoke certificate for ${name} in context of ${server}":
-    command  => ". ./vars && ./revoke-full ${name} ; test $? -eq 2 && touch revoked/${name}",
+    command  => ". ./vars && ./revoke-full ${name}; echo \"exit $?\" | grep -qE '(error 23|exit (0|2))' && touch revoked/${name}",
```

Comment:
What kind of problem does this solve? Doesn't your version of `revoke-full` return an exit code of 2?